src/styles/index.scss

```diff
@@ -1,3 +1,88 @@
-body {
+$color-border: #000;
+$color-minute-hand: #0700ff;
+$color-second-hand: #2c8000;
+$color-center: #f6a603;
+$size-stopwatch: 80vmin;
+$size-center: 5vmin;
+$width-minute-hand: 3vmin;
+$length-minute-hand: 20vmin;
+$width-second-hand: 1.5vmin;
+$length-second-hand: 38vmin;
+$duration-normal: 60s;
+$duration-speed-up: 10s;
+$duration-minute-hand: 3600s;
+$duration-minute-hand-speed-up: 600s;
```

Comment:
Move variables to the separate file

Reply:
Why?

Reply:
It's better to store variables in separate file to make the files cleaner.